server/gc/src/main/java/org/apache/accumulo/gc/GarbageCollectWriteAheadLogs.java

```diff
@@ -326,7 +333,8 @@ private Map<UUID,TServerInstance> removeEntriesInUse(Map<TServerInstance,Set<UUI
           }
         }
       }
-    }
+    });
+    store.close();
```

Comment:
Should this be in a finally block? A lot of errors could happen in the prior block.